conjur/list/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
+List module
+
+This metafile includes all the functionality that will be exposed
+when you install this module
+"""
+from .list_data import ListData
+# from .login_logic import LoginLogic
+from .list_controller import ListController
```

Comment:
I really like this and I didn't knew that the way it works before you showed me.